README.md

```diff
@@ -32,7 +48,7 @@ CCX notification writer service
  report     | character varying           | not null
  updated_at | timestamp without time zone | not null
 Indexes:
-    "new_reports_pkey" PRIMARY KEY, btree (org_id, cluster)
+    "new_reports_pkey" PRIMARY KEY, btree (org_id, cluster, updated_at)
```

Comment:
Shouldn't we have a similar primary key for the `reported` table (including `updated_at`value)?

Reply:
hmm we just need to know that some issue has been reported at given time, but it will be rather `notification_type`. We are not as far ATM :/